src/transformers/models/mctc/configuration_mctc.py

```diff
@@ -0,0 +1,161 @@
+# coding=utf-8
+# Copyright 2022 Chan Woo Kim and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""MCTC model configuration"""
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+MCTC_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "mctc-large": "https://huggingface.co/mctc-large/resolve/main/config.json",
+    # See all mCTC models at https://huggingface.co/models?filter=mctc
+}
+
+
+class MCTCConfig(PretrainedConfig):
+    r"""
+    This is the configuration class to store the configuration of a [`~MCTCModel`]. It is used to instantiate an mCTC
+    model according to the specified arguments, defining the model architecture. Instantiating a configuration with the
+    defaults will yield a similar configuration to that of the mCTC [mctc-large](https://huggingface.co/mctc-large)
+    architecture.
+
+    Configuration objects inherit from [`PretrainedConfig`] and can be used to control the model outputs. Read the
+    documentation from [`PretrainedConfig`] for more information.
+
+
+    Args:
+        vocab_size (`int`, *optional*, defaults to 8065):
```

Comment:
think the docstrings need to be updated here :-)

Reply:
updated!